CDV-diner-project/Dishes.json

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
Według mnie nie powinien to być obiekt tylko tablica, aby łatwo można się odwoływać. W tym wypadku musisz znać wszystkie kategorie z góry w kodzie.

Reply:
W sensie teoretycznie możesz jakoś pobrać klucze, i z tym się bawić. Ale to nie powinno być tak rozwiązane